reana_workflow_controller/rest/workflows.py

```diff
@@ -238,17 +248,18 @@ def get_workflows(paginate=None):  # noqa
                 # Skip workflow if type is interactive and there is no session
                 elif type_ == "interactive":
                     continue
+            empty_disk_usage = {
+                "human_readable": "",
+                "raw": -1,
+            }
             if verbose:
-                try:
-                    disk_usage_info = workflow.get_workspace_disk_usage(
-                        summarize=True, block_size=block_size
-                    )
-                    if disk_usage_info:
-                        workflow_response["size"] = disk_usage_info[0]["size"]
-                    else:
-                        workflow_response["size"] = "0K"
-                except ValueError:
-                    workflow_response["size"] = "0K"
+                workflow_response["size"] = (
+                    workflow.get_quota_usage()
+                    .get("disk", {})
+                    .get("usage", empty_disk_usage)
+                )
+            else:
+                workflow_response["size"] = empty_disk_usage
```

Comment:
This is not pretty, but we [cannot specify response properties to be nullable with OpenAPI2](https://stackoverflow.com/questions/48111459/how-to-define-a-property-that-can-be-string-or-null-in-openapi-swagger)... If you have any idea on how to do it better, please share.

Reply:
Nothing comes to my mind so far.